kitchen.recipes.yml

```diff
@@ -397,3 +397,22 @@ suites:
     attributes:
       dependencies:
         - recipe:aws-parallelcluster-install::chrony
+  - name: slurm_installed
+    run_list:
+      - recipe[aws-parallelcluster::add_dependencies]
+      - recipe[aws-parallelcluster-slurm::install_slurm]
+    verifier:
+      controls:
+        - slurm_installed
+        - slurm_user_and_group_created
+        - slurm_licence_configured
+        - slurm_library_shared
+    attributes:
+      dependencies:
+        - recipe:aws-parallelcluster::test_dummy
+        - recipe:aws-parallelcluster-install::directories
+        - resource:package_repos
+        - resource:install_packages
+        - recipe:aws-parallelcluster-install::python
+        - recipe:aws-parallelcluster-slurm::install_jwt
+        - recipe:aws-parallelcluster-slurm::install_pmix
```

Comment:
munge is also required for the installation of Slurm. Please add it to the dependencies.

Reply:
Good catch. 
Adding ...